File: week1_day3.js
function acronymMaker(string) {
    string = string.replace(/\s+/g, " ").trim();
    let stringArr = string.split(" ");
    let acronym = "";
    if(stringArr[0] === "") {
        return "EZ acronym";
    }
    for(let i=0; i<stringArr.length; i++) {
        acronym += stringArr[i][0];
    }
    return acronym.toUpperCase();
}

console.log(acronymMaker("Behold My Glory"));
console.log(acronymMaker("what is life?"));
console.log(acronymMaker("   Woah   those spaces  "));
console.log(acronymMaker(""));

File: week1_day3.py
def acronymMaker(string):
    stringArr = string.split()
    acronym = ""
    if len(stringArr) == 0:
        return "EZ acronym"
    for element in stringArr:
        acronym += element[0]

    return acronym.upper()

print(acronymMaker("Behold My Glory"))
print(acronymMaker("what is life?"))
print(acronymMaker("   Woah   those spaces  "))
print(acronymMaker(""))
